authors/apps/articles/urls.py

```diff
@@ -1,10 +1,20 @@
 from django.urls import path
 
-from authors.apps.articles.views import CreateArticlesAPIView, RetrieveUpdateDeleteArticleAPIView
+from authors.apps.articles.views import (
+    CreateArticlesAPIView, RetrieveUpdateDeleteArticleAPIView, LikesView
+)
+from .models import Articles, LikeDislike
 
 app_name = 'articles'
 
 urlpatterns = [
     path('articles/', CreateArticlesAPIView.as_view(), name='articles'),
-    path('article/<slug:slug>/', RetrieveUpdateDeleteArticleAPIView.as_view(), name='article')
-]
\ No newline at end of file
+    path('article/<slug:slug>/',
+         RetrieveUpdateDeleteArticleAPIView.as_view(), name='article'),
+    path('article/<slug:slug>/like/',
```

Comment:
change urls to follow pattern agreed on by the group

Reply:
I'll rebase for this